workflow/tests/hpxml_translator_test.rb

```diff
@@ -502,6 +502,17 @@ def skip_utility_bill_warning(err_line)
       end
       return skip_warning
     end
+
+    def skip_electricity_production_warning(err_line)
```

Comment:
Why are we skipping these warnings as opposed to preventing them from occurring?

Actually, same question for `skip_utility_bill_warning` above. What warnings are we trying to skip there, and why can't we prevent them from occurring altogether?

Reply:
It looks like the `skip_utility_bill_warning` was added in https://github.com/NREL/OpenStudio-HPXML/pull/955, which suggests that it was added specifically for ReportUtilityBills (and not ReportSimulationOutput). Now that we have a `HPXML::has_fuel` method, maybe we can avoid these warnings now?

Reply:
i.e. make requests based on existence of fuel?

Reply:
Is this out of scope for this PR?

Reply:
Yes.

Reply:
We can tackle the utility bill  later, that's fine. But I'd still like to know why we're getting additional warnings in this PR such that you had to add a new `skip_electricity_production_warning`.

Reply:
For error checking I'm now subtracting storage from production, so this is no longer an issue. (It was because I was requesting Photovoltaic and Cogeneration variables regardless of what was in the model.)

Reply:
Nice. I moved the utility bill warnings to a separate issue. https://github.com/NREL/OpenStudio-HPXML/issues/1210